DisCatSharp/Entities/Interaction/DiscordInteraction.cs

```diff
@@ -70,7 +70,7 @@ public DiscordGuild Guild
 	/// <summary>
 	/// Gets the channel that invoked this interaction.
 	/// </summary>
-	[JsonIgnore]
+	[JsonIgnore] // TODO: Is now also partial "channel"
 	public DiscordChannel Channel
 		 => (this.Discord as DiscordClient).InternalGetCachedChannel(this.ChannelId) ?? (DiscordChannel)(this.Discord as DiscordClient).InternalGetCachedThread(this.ChannelId) ?? (this.Guild == null ? new DiscordDmChannel { Id = this.ChannelId, Type = ChannelType.Private, Discord = this.Discord } : new DiscordChannel() { Id = this.ChannelId, Discord = this.Discord });
```

Comment:
wouldn't it be nicer to maybe rewrite this so it's easier to read?

Reply:
true, but for this pr out of scope